File: swagger-codegen/csharp-dotnet2/src/main/CsharpDotNet2/samsara/Model/TripResponseStartCoordinates.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace samsara.Model {

  /// <summary>
  /// Start (latitude, longitude) in decimal degrees.
  /// </summary>
  [DataContract]
  public class TripResponseStartCoordinates {
    /// <summary>
    /// Gets or Sets Latitude
    /// </summary>
    [DataMember(Name="latitude", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "latitude")]
    public double? Latitude { get; set; }

    /// <summary>
    /// Gets or Sets Longitude
    /// </summary>
    [DataMember(Name="longitude", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "longitude")]
    public double? Longitude { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TripResponseStartCoordinates {\n");
      sb.Append("  Latitude: ").Append(Latitude).Append("\n");
      sb.Append("  Longitude: ").Append(Longitude).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: openapi-generator/dart/lib/model/trip_response_start_coordinates.dart
part of openapi.api;

class TripResponseStartCoordinates {
  
  double latitude = null;
  
  double longitude = null;
  TripResponseStartCoordinates();

  @override
  String toString() {
    return 'TripResponseStartCoordinates[latitude=$latitude, longitude=$longitude, ]';
  }

  TripResponseStartCoordinates.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['latitude'] == null) {
      latitude = null;
    } else {
          latitude = json['latitude'];
    }
    if (json['longitude'] == null) {
      longitude = null;
    } else {
          longitude = json['longitude'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'latitude': latitude,
      'longitude': longitude
    };
  }

  static List<TripResponseStartCoordinates> listFromJson(List<dynamic> json) {
    return json == null ? new List<TripResponseStartCoordinates>() : json.map((value) => new TripResponseStartCoordinates.fromJson(value)).toList();
  }

  static Map<String, TripResponseStartCoordinates> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, TripResponseStartCoordinates>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new TripResponseStartCoordinates.fromJson(value));
    }
    return map;
  }
}

